### PR TITLE
Managed hosts

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)

### CODE REVIEW

**Review by bangelic — COMMENTED** [review]
I also found instances of `managed hosts` in the following:

common/modules/con_host-substatus-overview.adoc
common/modules/con_puma-workers-and-threads-recommendations.adoc
common/modules/proc_attaching-red-hat-subscriptions-to-content-hosts.adoc
common/modules/proc_enabling-power-management-on-managed-hosts.adoc
common/modules/proc_tuning-with-predefined-profiles.adoc
common/modules/proc_using-novnc-to-access-virtual-machines.adoc
common/modules/ref_discovered-settings.adoc
doc-Planning_for_Project/topics/Capsule_Server_Overview.adoc
doc-Planning_for_Project/topics/Deployment_Considerations.adoc
doc-Planning_for_Project/topics/Glossary.adoc
doc-Planning_for_Project/topics/Introduction.adoc

**Review by Lennonka — COMMENTED** [review]
As Brian says ^^ the replacement possibly isn't complete. The question remains whether some of those occurences aren't cases, in which the special meaning of "managed host" is appropriate. Better double check with a SME?

File: guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ Adding a security exception in the browser is not enough for using noVNC.
 For more information, see {AdministeringDocURL}Importing_the_Katello_Root_CA_Certificate_admin[Installing the Katello Root CA Certificate] in _{AdministeringDocTitle}_.
 
 .Procedure
-. On your managed host, configure the firewall to allow VNC service on ports 5900 to 5930.
+. On your host, configure the firewall to allow VNC service on ports 5900 to 5930.
```
Comment (ekohl) [inline guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc:20]: This is on the server, so should be:
```suggestion
. On your {ProjectServer}, configure the firewall to allow VNC service on ports 5900 to 5930.
```

**Review by ekohl — CHANGES_REQUESTED** [review]
I only checked the current changes, didn't dig for further missing ones.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ A sub-status monitors only part of a host's capabilities.
 Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
 There can be more sub-statuses depending on which plugins you add to your {Project}.
 
-The *build* sub-status is relevant for managed hosts and when {Project} runs in unattended mode.
+The *build* sub-status is relevant for hosts and when {Project} runs in unattended mode.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc]: OK, thanks. Applied.

File: guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 = Configuring a Host to Use the Pull Client
 
 For {SmartProxies} configured to use `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
-Managed hosts do not require an SSH connection to their {SmartProxyServer}.
+Hosts do not require an SSH connection to their {SmartProxyServer}.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc]: Fixed.

File: guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_using-novnc-to-access-virtual-machines.adoc]: Thanks, applied.

**Review by maximiliankolb — COMMENTED** [review]
Thanks Ewoud. I have applied your feedback in three separate commits.

**Review by ekohl — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.9: :heavy_check_mark: 
   1c739553f..9334ae602  3.9 -> 3.9

**Pull request merged**